lib/node/nodes/data-observatory-measure.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var dot = require('dot');
+dot.templateSettings.strip = false;
+
+var Node = require('../node');
+
+var TYPE = 'data-observatory-measure';
+var PARAMS = {
+    source: Node.PARAM.NODE(Node.GEOMETRY.POLYGON),
```

Comment:
I've just followed the provided description `Calculate Data Observatory measure within a Polygon`. But there isn't any  constraint. ;)